persistence.repositories.sequelize/src/flow_node_instance_repository.ts

```diff
@@ -254,14 +241,45 @@ export class FlowNodeInstanceRepository implements Repositories.IFlowNodeInstanc
     });
   }
 
+  public async reset(flowNodeInstanceId: string): Promise<void> {
+    const queryParams: FindOptions = {
+      where: {
+        flowNodeInstanceId: flowNodeInstanceId,
+        state: DataModels.FlowNodeInstance.FlowNodeInstanceState.error,
+      },
+    };
+
+    const flowNodeInstance = await FlowNodeInstanceModel.findOne(queryParams);
+
+    if (!flowNodeInstance) {
+      throw new Error(`The FlowNodeInstance with ID ${flowNodeInstanceId} has no error state.`);
```

Comment:
```suggestion
      throw new NotFoundError(`FlowNodeInstance with ID ${flowNodeInstanceId} not found.`);
```

`NotFoundError` kommt aus `@atlas-engine/error_provider`